tamr_unify_client/project/attribute_mapping/collection.py

```diff
@@ -55,3 +55,14 @@ def create(self, creation_spec):
         """
         data = self.client.post(self.api_path, json=creation_spec).successful().json()
         return AttributeMapping(data)
+
+    def delete_by_resource_id(self, resource_id):
+        """delete an attribute mapping using its Resource ID.
+        :param resource_id: the resource ID of the mapping to be deleted.
+        :type resource_id: str
+        :returns: the status code (200, 400, etc.)
+        :rtype: HTTP status code
```

Comment:
What you're returning here isn't just the status code, but the response itself, so type :class:\`requests.Response\`